src/sentry/migrations/0742_backfill_broken_monitor_notification_setting_option.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 5.0.6 on 2024-07-17 08:41
+
+from django.apps.registry import Apps
+from django.db import migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def backfill_broken_monitor_notification_setting_option(
+    apps: Apps, schema_editor: BaseDatabaseSchemaEditor
+) -> None:
+    NotificationSettingOption = apps.get_model("sentry", "NotificationSettingOption")
+
+    # Migrate all nudge settings to the new broken cron settings
+    approval_settings = NotificationSettingOption.objects.filter(type="approval")
+    for setting in RangeQuerySetWrapperWithProgressBar(approval_settings):
+        NotificationSettingOption.objects.get_or_create(
+            user_id=setting.user_id,
+            scope_type=setting.scope_type,
+            scope_identifier=setting.scope_identifier,
+            type="brokenMonitors",
+            defaults={"value": setting.value},
+        )
```

Comment:
Do we need to remove the `approval` type setting?

Reply:
nah, the approval setting still exists and will control other classes of notifications, just wanted to split out broken monitor emails into its own category with this migration